src/parser/flex_lexer.l

```diff
@@ -131,6 +131,8 @@ OUTER		TOKEN(OUTER)
 RIGHT		TOKEN(RIGHT)
 TABLE		TOKEN(TABLE)
 UNION		TOKEN(UNION)
+INTERSECT	TOKEN(INTERSECT)
+EXCEPT		TOKEN(EXCEPT)
```

Comment:
Whitespace

Reply:
The file is tab formated...I'm not quite sure how to best aligne it. Currently it's aligned for tab_width >= 4. 

Or do you mean something else for Whitespace?

Reply:
Weird. Now it looks correct. No idea what github showed me. nevermind